### DISCUSSION TITLE
1.9.2 support for PS2/3 pressure buttons? need to activate?

### DISCUSSION THREAD

**Discussion by patrikmckane** [darthcloud/BlueRetro#917] [General]
Hi i see that you added support PS2/3 pressure buttons. How exactly it will work? Or it needs to be someewhere turned on? i updated to 1.9.2, i use genuine ps3 controller but it does not work. the blueretro PS2 is all default i changed nothin in the setting. Thanks for answer

**Comment by ButtonBasher92** [comment]
Should work the moment you connect a DS3 controller. In Advanced Config scroll down to Mapping Config and change SRC Lable to PS3 and DST Lable to PlayStation or PS2 I forgot what it's called and save. see if that maybe works.

If not then there may be a high chance your PlayStation 3 controller is fake. some are virtually impossible to tell apart from the original without opening them up and looking at the PCB and battery.

**Reply by patrikmckane** [reply]
Controller is definitely genuine. i try the settings.

**Reply by ButtonBasher92** [reply]
Okay might also prove helpful to change System to "PS2" under Global Config. I use DS3 controllers on my PS2 and this will mimic my settings and my pressure sensitive buttons all work.

**Reply by patrikmckane** [reply]
Ok it is working i changed all 3 settings what you suggested. Its not perfect it is too sensitive compared to og ps2 controller or the logitech wireless but atleast it is working. Thanks

**Reply by ButtonBasher92** [reply]
Great! No Problem. I agree, it takes some getting used to when moving from a ps2 controller, they feel quite different to one another, analogue sticks especially. But I think it's down to the PS3 controller itself and not software related.
